--- setup.py
+++ setup.py
@@ -19,7 +19,6 @@
 
 __revision__ = '$Id: setup.py,v 1.15 2005/01/03 18:43:53 alf Exp $'
 
-from __future__ import nested_scopes
 import os
 import sys
 import shutil
--- propagation.py
+++ propagation.py
@@ -18,7 +18,6 @@
 
 __revision__ = '$Id: propagation.py,v 1.36 2005/07/13 08:48:22 ludal Exp $'
 
-from __future__ import generators
 from logilab.constraint.interfaces import DomainInterface, ConstraintInterface
 import bisect, operator
 from time import strftime
