--- /home/wrstl/temp/python-ogre-build/python-ogre/PythonOgreConfig_posix.py	2008-04-21 07:52:02.000000000 +0200
+++ PythonOgreConfig_posix.py	2008-04-21 20:58:10.000000000 +0200
@@ -11,16 +11,16 @@
 MAC_cflags = ' --gccxml-cxxflags "-DCF_OPEN_SOURCE -D_POSIX_C_SOURCE -isysroot /Developer/SDKs/MacOSX10.4u.sdk"'
 
 # the base of the /usr/... dircetory structure that we are using
-ROOT_DIR = os.path.join(BASE_DIR,'root')
+ROOT_DIR = '/'
 LOCAL_LIB = os.path.join(ROOT_DIR,'usr/lib')
 LOCAL_INCLUDE = os.path.join(ROOT_DIR, 'usr/include')
 
 ## path to the root of your boost dir, should have 'libs', 'boost' subdirectories
-PATH_Boost = os.path.join(LOCAL_INCLUDE, 'boost-1_34_1')
+PATH_Boost = LOCAL_INCLUDE
 ## Path to your boost_pythonxxxx lib file
 PATH_LIB_Boost = LOCAL_LIB
 ## and the name of the boost python library
-LIB_Boost = 'libboost_python-gcc41-1_34_1'
+LIB_Boost = 'libboost_python'
 
 # in Linux we need to code in the Relative path for the library
 RPATH=os.path.join('\\$$ORIGIN',os.pardir,os.pardir, 'lib') #+ '\'' 
@@ -29,7 +29,7 @@
 gccxml_bin = os.path.join(BASE_DIR,'gccxml-build/bin')
 gccxml_bin = os.path.join(ROOT_DIR,'bin')
 # and the Py++ directory as sometimes we need access to the code repository there
-pyplusplus_install_dir = os.path.join(BASE_DIR,'pygccxml')
+pyplusplus_install_dir = '.' # SHOULD NOT BE NECESSARY os.path.join(BASE_DIR,'pygccxml')
 
 SDK = False
 if os.sys.platform == 'darwin':  # we use the pre built sdk for OSX
@@ -46,7 +46,7 @@
     PATH_CEGUI =        os.path.join(BASE_DIR, 'ogrenew','Dependencies')
     PATH_OIS =          os.path.join(BASE_DIR, 'ogrenew','Dependencies')
     
-PATH_Newton =       os.path.join(BASE_DIR, 'newtonSDK','sdk')
+PATH_Newton =       LOCAL_INCLUDE # os.path.join(BASE_DIR, 'newtonSDK','sdk')
 PATH_ODE =          os.path.join(BASE_DIR, 'ode-0.9')
 PATH_OGG =          os.path.join(BASE_DIR, 'libogg-1.1.3')
 PATH_VORBIS=        os.path.join(BASE_DIR, 'libvorbis-1.2.0')
@@ -107,7 +107,7 @@
 PATH_LIB_OgreAL =               os.path.join( LOCAL_LIB ) #PATH_OGREAL, 'lib/Release' )
 PATH_LIB_betagui =              PATH_betagui
 PATH_LIB_quickgui =             PATH_quickgui
-# PATH_LIB_NxOgre=                os.path.join(LOCAL_LIB )
+PATH_LIB_NxOgre=                os.path.join(LOCAL_LIB )
 PATH_LIB_PhysX =                os.path.join(LOCAL_LIB, 'PhysX','v2.7.3')
 PATH_LIB_Bullet = 		        os.path.join( LOCAL_LIB )
 PATH_LIB_Theora=                os.path.join(PATH_Theora, 'bin', 'Release')
